### ISSUE TITLE
project logo

### ISSUE THREAD

**Issue by declann** [calculang/calculang#51]
Starting concept:

![logo0](https://user-images.githubusercontent.com/353214/151708631-43d9d5f9-aedc-488e-8572-62f43cf94a08.svg)

Requirements:

Simple, colorful, and made using free software tools (Inkscape is brilliant).

**Comment by declann** [comment]
And logo should evolve over time, I think of above as an OK concept starting point, recognizing being coding-tied now.

But numbers or representations of visuals are probably better features in the grand scheme of things versus code representations (the lines above).

Other possible prop ideas: magnifying glass?

Feel free to comment or send ideas/improvements.

**Comment by declann** [comment]
I set my twitter profile to the concept SVG.

https://twitter.com/calculang

In small-form it looks like a postcard and the features are lost, need to be better emphasized (color/spacing/sizes).

**Comment by declann** [comment]
Update, but already going too complex imo.

![logo1](https://user-images.githubusercontent.com/353214/151713677-4348df27-a9d3-4e59-9bd6-045fc060b658.svg)

**Comment by declann** [comment]
![logo2](https://user-images.githubusercontent.com/353214/151714206-af5a90a1-81c9-4665-8f2f-4d4ee41caebb.svg)

**Comment by declann** [comment]
![logo3](https://user-images.githubusercontent.com/353214/151714416-3589802f-b4da-4f53-a6b8-7500b2d2d9ae.svg)

**Comment by declann** [comment]
I think:

x looks like there is something wrong
division looks like + when zoomed way out (twitter icon in tweets)
So preferring - which is also simpler:
![logo4](https://user-images.githubusercontent.com/353214/151716154-8aaf0443-f756-43fb-9f50-65b5ab879c07.svg)

**Comment by declann** [comment]
Landed on twitter:

https://twitter.com/calculang/status/1487896873394393089

Feedback welcome, tweaks will be wanted in due course, but for a while may slowly roll into documentation and profiles.

**Comment by ProminentChrystal** [comment]
You're making it complicated, remember a logo is usually really tiny when displayed on most places. Like 50x50 or 100x100. Most of the small details you added won't even be visible at that size. Also try sticking to fewever colors (Like 1-3).

The green isn't bad but the palette as a whole doesn't really work together.

Here are two palattes with green you might like:
https://colorhunt.co/palette/d3eca7a1b57db3303019282f
https://colorhunt.co/palette/faf1e6fdfaf6e4efe7064420

Here's a cool gradient in case you're considering gradients (it doesn't have green in it tho): https://uigradients.com/#Bourbon

Also how about using text like "Cal()" as the logo? You could just use it as is or you could even put it on the text bubble thing.

**Comment by declann** [comment]
Hey, thanks for the feedback! 😃 

> You're making it complicated, remember a logo is usually really tiny when displayed on most places. Like 50x50 or 100x100. Most of the small details you added won't even be visible at that size.

Agreed, I'm working on a website and just converted my icon to a favicon: so I can see this problem very clearly!

The only reference I was using in designing logo above was my twitter profile: https://twitter.com/calculang

I emphasized the features enough to barely pass at that size; there is a real problem at smaller sizes.

I think I need a **responsive icon** e.g. just show speech bubble and equals symbol at small sizes. This form factor is clearly important for favicon and especially important because its also relevant for a browser extension button!

If someone wants to tweak logo/design small logos under appropriate licensing that I can pick off, I'll consider that. But an important factor is to include at least some of the [meaning of the logo](https://twitter.com/calculang/status/1487896873394393089). That can be fewer symbols maybe (necessary at small sizes), it can be different symbols, but I don't think Calc() fits for example.

About colors, I totally made this up as I went and it deserves more attention! Cheers for the references. I think its hard to get down to 2/3 colors, but maybe worth experimenting here.
The separate colors in the lines which represent code are used to create a hint about approachability, because bland code is a bad symbol.
I've thought about replacing the code symbol altogether e.g. using a bar chart or graph symbol instead, but that's not something I wanted for this iteration.

There will be other iterations, and I agree with your feedback about palette and detail at small sizes, so these need to be high priorities in future development of logo!
